src/Table/elements.js

```diff
@@ -1,197 +1,107 @@
- *
- * @returns {string} - The value of the CSS width attribute.
- */
-const getCellWidth = width => (width !== parseInt(width).toString() ? width : 'initial');
-
-const cellAlignToJustify = {
-  left: 'flex-start',
-  center: 'center',
-  right: 'flex-end',
-};
-
-const Cell = styled.div`
-  box-sizing: content-box;
-  display: flex;
-  justify-content: ${({ align }) => cellAlignToJustify[align || 'center']};
-  align-items: center;
-
-  height: 6rem;
-  width: ${({ width }) => getCellWidth(width || 'auto')};
-  flex: ${({ width }) => getCellFlex(width || '1')};
-
-  padding: 0 1rem;
-
-  color: ${({ theme: { palette } }) => palette.mediumGrey};
-`;
+// Table Header
+export const TableHeader = styled.thead`
+  font-size: ${({
+    theme: {
+      fonts: { size },
+    },
+  }) => size.default};
+  font-weight: ${({
+    theme: {
+      fonts: { weight },
+    },
+  }) => weight.thick};
+  line-height: 1.8rem;
 
-export const HeaderCell = styled(Cell).attrs({
-  data: ({ j }) => `header-cell-${j}`,
-})`
-  display: flex;
+  text-transform: uppercase;
 
-  background: transparent;
+  color: ${({ theme: { palette } }) => palette.darkGrey};
 `;
 
-export const BodyCell = styled(Cell).attrs({
-  data: ({ j }) => `body-cell-${j}`,
-})`
-  font-size: $({theme: {fonts: size}} => size.medium);
-  line-height: $({theme: {fonts: size}} => size.medium);
-`;
+export const TableHeaderCell = styled.th`
+  height: 4.4rem;
+  &:first-child {
+    padding-left: 3rem;
+  }
+  &:last-child {
+    padding-right: 3rem;
+  }
 
-export const HeaderCellTitle = styled.div`
-  display: flex;
-  align-items: center;
+  text-align: ${({ align }) => align || 'left'};
 `;
 
-export const HeaderSortingContainer = styled.div.attrs({
-  data: 'sorting',
-})`
+export const HeaderSortingContainer = styled.div`
   display: flex;
-  justify-content: center;
   align-items: center;
 
-  width: 2.5rem;
-  height: 100%;
-
-  margin-right: ${({ theme: { dimensions } }) => dimensions.small};
-
-  cursor: pointer;
-`;
-
-export const HeaderSortingGhost = styled.div`
-  width: 2.5rem;
-  height: 1rem;
-
-  margin-left: ${({ theme: { dimensions } }) => dimensions.small};
-
   ${({ align }) =>
     align === 'right' &&
     css`
-      display: none;
+      justify-content: flex-end;
     `};
+
+  cursor: pointer;
 `;
 
 export const HeaderLabel = styled.div`
-  font-weight: ${({
-    theme: {
-      fonts: { weight },
-    },
-  }) => weight.thick};
+  margin-right: 0.8rem;
+  user-select: none;
 `;
 
-const Row = styled.div`
-  display: flex;
-  align-items: center;
-
-  width: 100%;
-  height: 6rem;
-
-  border-bottom: 1px solid hsl(0, 0%, 94%);
-
-  &:last-child {
-    border: none;
+// Table Body
+export const Body = styled.tbody`
+  td {
+    height: 6rem;
+    font-weight: ${({
+      theme: {
+        fonts: { weight },
+      },
+    }) => weight.thick};
+  }
+  td:first-child {
+    padding-left: 3rem;
+  }
+  td:last-child {
+    padding-right: 3rem;
```

Comment:
You could maybe put the child pseudo-elements in td `&:first-child` `&:last-child`